app/Services/API/User/PostService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace App\Services\API\User;
+
+
+use App\Models\Post;
+
+class PostService
+{
+    /**
+     * @return array
+     */
+    public function getRecentPost(int $day = 1): array
+    {
+        $posts = Post::with('user', 'game')->whereRaw('datediff(now(), created_at) <= ?', [$day])->get();
```

Comment:
Tính toán datetime bằng php rồi truyền string vào cho mysql thôi nhé, ko nên để mysql tính toán